tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -8028,3 +8028,165 @@ TEST_F(EnergyPlusFixture, StatFileCharacterMatching)
     parseStatLine(koppenLineNoDots, lineTypeReturn, desCondLinePassed, htgDesignLinePassed, clgDesignLinePassed, isKoppen);
     EXPECT_EQ((int)StatLineType::KoppenLine, (int)lineTypeReturn);
 }
+
+TEST(OutputReportTabularTest, GetDelaySequencesSurfaceOrder_test)
```

Comment:
New unit test calls `GetDelaySequences` for a zone with four surfaces, saves the result, then reorders the surfaces, and expects the same result. Without the fix to `GetDelaySequences`, this test fails.